notebooks/monitoring.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+echo ==================================
+echo =========== MONITORING ===========
+echo ==================================
+echo --- General Information ---
+# number of cores
+echo \#CPU: $(nproc)
+
+# multiply by 10^-6 to convert KB to GB
+echo Total Memory: $(cat /proc/meminfo | grep MemTotal | awk 'BEGIN { FS=" " } ; { print $2/1000000 }')G
```

Comment:
There's some common code between here and `get_mem_usage` which could be pulled into a `get_total_mem` function